code/modules/modular_computers/file_system/programs/ntnrc_client.dm

```diff
@@ -36,6 +36,12 @@
 			var/message = reject_bad_text(params["message"])
 			if(!message)
 				return
+			//yogs start
+			if(isnotpretty(message))
+				to_chat(usr, "<span class='notice'>Your fingers slip. <a href='https://forums.yogstation.net/index.php?pages/rules/'>See rule 0.1.1</a>.</span>")
```

Comment:
Its just rule 0.1, and the link is https://forums.yogstation.net/help/rules/

Reply:
I copied most of this from what's in the say code; consider updating that text as well.